package.json

```diff
@@ -100,6 +100,7 @@
     "stream-browserify": "^3.0.0",
     "sucrase": "^3.9.5",
     "typescript": "^4.1.2",
+    "updot": "^1.1.7",
```

Comment:
We discussed this before, and `updot` should only be a dependency for the GUI, where `yarn link` doesn't work. If you prefer the `updot` workflow in general, then it's up to you to install it globally on your computer, such as with with `npm i -g updot` so you can use it wherever you like. Otherwise, the normal NPM / yarn symlink workflows work fine in these other places.

Reply:
ok. thats fair. I rebased the PR to exclude that commit.